osu.Game.Tests/Visual/SongSelectV2/TestSceneSongSelectFiltering.cs

```diff
@@ -196,6 +196,19 @@ public void TestSorting()
             AddStep(@"Sort by Source", () => config.SetValue(OsuSetting.SongSelectSortingMode, SortMode.Source));
         }
 
+        [Test]
+        public void TestHardDeleteHandledCorrectly()
```

Comment:
What about standard / soft delete? "Hard delete" rarely–if ever–happens and I'm not sure the relevance of this here.

Reply:
I just noticed I missed some tests existing in previous test scene regarding this. Will port them over.